web-components/src/components/combobox/ComboBox.test.ts

```diff
@@ -1645,12 +1624,12 @@ describe("Combobox Component", () => {
         `
       );
       el.expanded = true;
-      const lists = comboBoxOptions.map((item, index) => {
-        return `<li id=${item} role="option" style="height:12px;">${item}</li>`;
+      const lists = comboBoxOptions.map((item, _index) => {
+        return `<div id=${item} class="md-combobox-option" role="option" style="height:12px;">${item}</div>`;
```

Comment:
since you are modifying this code, can you remove hardcoded style style="height:12px;"?

Reply:
This is unit test